semantiva/logger/logger.py

```diff
@@ -2,6 +2,10 @@
 import logging
 from typing import Optional
 
+DEFALT_FORMATTER = logging.Formatter(
```

Comment:
```suggestion
DEFAULT_FORMATTER = logging.Formatter(
```

But it is not good to accept here because it wil not propagate

Reply:
done